linera-execution/src/execution_state_actor.rs

```diff
@@ -395,12 +397,20 @@ where
                     ExecutionError::UnauthorizedHttpRequest(url)
                 );
 
-                let response = Client::new()
+                #[cfg_attr(web, allow(unused_mut))]
+                let mut request = Client::new()
```

Comment:
So on a web client there won't be any timeouts?

Reply:
Only the default browser timeouts. Unfortunately we can't add timeouts with `reqwest` for Wasm, we'd need to use Javascript APIs directly :(